src/index.html

```diff
@@ -51,13 +56,12 @@
     </script>
 
     <style>
-      body {
-        background: #234a61;
-      }
-
-      .loading footer {
-        display: none;
-      }
+    html, body {
+      background-color: #152531;
```

Comment:
do we still need this here if we also moved it up now?

Reply:
Not sure, really. I left it in, since it could be that the CSS we load in-between resets the color again, no?

Reply:
only if we explicitly have something in there that sets it to white or some other color again.. other wise, it's 'cascading style sheets', and it cascades into eachother.